app/src/main/res/layout/profile_reset_pin_activity.xml

```diff
@@ -62,14 +62,11 @@
 
           <TextView
             android:id="@+id/profile_reset_pin_main"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
```

Comment:
Body uses different layout width/height properties than this--is it the correct style to be using? This TextView also didn't define an explicit font family before.

Reply:
Fixed style for this view.